dist/Dockerfile.build/Dockerfile

```diff
@@ -11,8 +11,8 @@ ENV HOME="/root"
 ENV PATH="${HOME}/.cargo/bin:${PATH}"
 
 # build: Rust stable toolchain + 2 previous versions
-RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.45.0 -y && \
-  rustup install 1.42.0 && \
+RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.46.0 -y && \
+  rustup install 1.46.0 && \
   rustup install 1.43.0 && \
   rustup install 1.44.0 && \
   rustup default 1.45.0
```

Comment:
```suggestion
  rustup default 1.46.0
```

Reply:
line 15 already contains 1.46.0. I used the minimal diff approach to bump here.

Reply:
I removed the `rustup default` command as it's redundant with the `--default-toolchain` argument given above.